### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by SachinRammoorthy**

### CODE REVIEW

**Comment by SachinRammoorthy** [comment]
@garg-anuj build failing because agreement hasn't been accepted? Please have a look.
https://travis-ci.org/jboss-outreach/rest-android/builds/315414030?utm_source=github_status&utm_medium=notification

**Comment by aashutoshrathi** [comment]
@SachinRammoorthy Travis Build failure is due to some other reasons we will fix it soon.

File: README.md [hunk 1]
```diff
@@ -6,78 +6,74 @@ This is a Android Client for testing Rest HTTP calls.
...
 
-9.Push to your repo git push origin branch-name
+9. Push to your repo `$ git push origin branch-name`
 
-10.Go into the Github repo and create a pull request explaining your changes
+10. Go into the Github repo and create a pull request explaining your changes.
 
-11.If you are requested to make changes, edit your commit using git commit
---amend, push again and the pull request will edit automatically
+11. If you are requested to make changes, edit your commit using `$ git commit --amend`, push again and the pull request will edit automatically
```
Comment (aashutoshrathi) [inline README.md]: Add a full stop at end, also add use `update` instead of `edit` for what happens to a PR

File: README.md [hunk 2]
```diff
@@ -6,78 +6,74 @@ This is a Android Client for testing Rest HTTP calls.
...
 
-12.You will need to add a message on the pull request notifying your changes to
-your reviewer
+12. You will need to add a message on the pull request notifying your changes to your reviewer.
```
Comment (aashutoshrathi) [inline README.md]: Better if we make it
```
12. You need to add a [commit message](https://github.com/jboss-outreach/gci/wiki/Commit-style-guidelines.) to your commit for letting the reviewer know that changes are made.
```

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi I have implemented the changes.

**Comment by aashutoshrathi** [comment]
@SachinRammoorthy Just check the full stops at the end once again, we missed some and also change your commit message to `docs: Updated README.md` and if possible squash these commits. 😄

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi encountered some problems while squashing so shifted to a new PR #57 with all the changes.

**Comment by aashutoshrathi** [comment]
😕 You should have asked here or on channel, it is not good practice to open multiple PRs for the same issue, please don't repeat it.

**Comment by SachinRammoorthy** [comment]
👍  Sorry for inconvenience

**Pull request closed without merging**